src/views/portals/outings-stats/OutingsStatsPart.vue

```diff
@@ -16,12 +16,52 @@
         <h2 class="title is-4" v-translate>height_diff_up</h2>
         <div ref="height_diff_up"></div>
       </div>
+
+      <div class="column">
+        <h2 class="title is-4" v-translate>height_diff_difficulties</h2>
+        <div ref="height_diff_difficulties"></div>
+      </div>
+    </div>
+
+    <div class="columns result-section">
+      <div class="column">
+        <h2 class="title is-4" v-translate>rock_free_rating</h2>
+        <div ref="rock_free_rating"></div>
+      </div>
+      <div class="column">
+        <h2 class="title is-4" v-translate>global_rating</h2>
+        <div ref="global_rating"></div>
+      </div>
```

Comment:
A more general comment: depending on activity selected, some graphs are not pertinent. I would rather hide them when it's the case.

Reply:
Actually no, because you can have several activity on one outing. And ratings can be missing, even if there is the relevant activity.

=> We can't show/hide graph based on activity, BUT we can hide graph if there is not data. It'll be the object of another PR :)